services/data/service_configs.py

```diff
@@ -0,0 +1,6 @@
+import os
+
+max_connection_retires = os.environ.get("MF_SERVICE_CONNECTION_RETRIES", 3)
+connection_retry_wait_time_seconds = os.environ.get("MF_SERVICE_CONNECTION_RETRY_WAITTIME_SECONDS", 1)
+max_startup_retries = os.environ.get("MF_SERVICE_STARTUP_RETRIES", 5)
+startup_retry_wait_time_seconds = os.environ.get("MF_SERVICE_STARTUP_WAITTIME_SECONDS", 1)
```

Comment:
Consider casting these to correct types. For example setting `MF_SERVICE_CONNECTION_RETRIES=1` would presumably fail in the `postgres_async_db` due to a string being passed to `range`